### PR TITLE
Update proto_ts_library to aspect/rules_ts

### PR DESCRIPTION
**Pull request by mrmeku**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -227,12 +223,47 @@ load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories")
...
+load("@rules_nodejs//nodejs:repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
+
+nodejs_register_toolchains(
+    name = "node",
+    node_version = DEFAULT_NODE_VERSION,
+)
+
+load("@aspect_rules_js//npm:npm_import.bzl", "npm_translate_lock")
+
+npm_translate_lock(
```
Comment (mrmeku) [inline WORKSPACE:258]: @pcj This part really should be part of `deps/ts_proto_deps.bzl`, as things are designed today, but I didn't think it was worth the complexity of updating the code generator. 

What do you think?

**Comment by pcj** [comment]
Thanks.  I'll take a look at this and https://github.com/stackb/rules_proto/commit/046f39d980f634d6af3e9b359d0fe587b1c63077#commitcomment-122688110 in the near future.

**Comment by pcj** [comment]
This for working on this @mrmeku , this is mostly good.

There are additional interactions as well as some code that was deleted in this PR (traditional js related outputs, routeguide example) should be preserved (some organizations still rely on these features).

I'm working on a PR on top of this, PR'd to your branch (https://github.com/mrmeku/rules_proto/pull/1)

**Comment by pcj** [comment]
This commit was merged in #339.

**Pull request merged**